client/Components/GameBoard/CardPile.jsx

```diff
@@ -19,6 +19,17 @@ class CardPile extends React.Component {
         };
     }
 
+    componentWillReceiveProps(props) {
+        let hasNewSelectableCard = props.cards && props.cards.some(card => card.selectable);
+        let didHaveSelectableCard = this.props.cards && this.props.cards.some(card => card.selectable);
+
+        if(!didHaveSelectableCard && hasNewSelectableCard) {
+            this.setState({ showPopup: true });
```

Comment:
It feels like this should be controlled at a higher level somehow, to handle the (admittedly rare) cases where cards are selectable in multiple piles. Per @DukeTax example in chat, Summer will potentially cause both the discard and dead piles to open simultaneously and overlapping. Ideally either only one would open or neither would open, leaving the player to click the one they want to choose from.

Reply:
I think I disagree with this somewhat.  Mainly because even in the case of multiple selection targets, how do we decide which should take priority?  Moving it up a level will involve a whole lot more passing around of state and components that have no business knowing about it being educated about it.

The other alternative is to drive it from the server, but I'm not sure that's a good idea either.

I personally think popping up the dead pile and discard pile is fine in the case of summer.  If the piles get in the way of each other than we can fix that with styling.